src/app/myworkflows/myworkflows.component.html

```diff
@@ -1,47 +1,61 @@
 <app-header>
-  <img src="../assets/images/dockstore/dockstore-workflows-purple.png">
-  My Workflows
+  <img src="../assets/images/dockstore/dockstore-workflows-purple.png"> My Workflows
   <div class="pull-right refresh-all-holder">
     <button type="button" class="refresh-all-button btn btn-primary" (click)="refreshAllWorkflows()">
       Refresh All Workflows
     </button>
   </div>
 </app-header>
 <div class="container padding-top">
+  <div class="row" *ngIf="orgWorkflows?.length == 0">
+    <div class="col-md-12">
+      <div class="alert alert-info" role="alert">
+        <p>
+          <span class="glyphicon glyphicon-info-sign"></span>
+          There are currently no workflows registered under this account, to add your first workflow, do one of the following:
+        </p>
+        <ul>
+          <li>
+            Go to 'Accounts' and link a Quay.io account that is set up to build Docker images from GitHub, Gitlab, or Bitbucket. Workflows will be synchronized
+            in the linking process.
```

Comment:
Workflows don't use quay.io, replace with 

"Press "Refresh all workflows" to create workflow stubs for all of your source control repositories. Browse to the workflow stubs and publish those that correspond to real workflows."